pages/common/vzdump.md

```diff
@@ -0,0 +1,28 @@
+# vzdump
+
+> Backup Utility for virtual machines and containers.
+> More information: <https://pve.proxmox.com/pve-docs/vzdump.1.html>.
+
+- Dump a guest virtual machine into default dump directory (usually `/var/lib/vz/dump/`), no snapshot:
+
+`vzdump {{vm_id}}`
+
+- Backup guest 101, 102, and 103:
```

Comment:
Is this an id number, a name, or something else?

Reply:
The number is the id of VM, in Proxmox, the VM start with 100, then 101

Reply:
```suggestion
- Backup guest virtual machine with id 101, 102, and 103:
```